### PR TITLE
saved db information in ssm

### PR DESCRIPTION
**Pull request by AgusPk**
# What's this PR do?
Added db port, name, host and secrets name to ssm

### Summary by CodeRabbit

- New Feature: Added four new AWS SSM parameters to securely store and manage critical database information such as address, name, port, and master user secret ARN. This enhancement improves the security and manageability of database-related configurations in our AWS environment.

### CODE REVIEW

File: modules/rds/ssm.tf [hunk 1]
```diff
@@ -0,0 +1,23 @@
+resource "aws_ssm_parameter" "db_instance_address" {
+  name  = "/${var.name}/db_instance_address"
+  type  = "StringList"
```
Comment (ulises-jeremias) [inline modules/rds/ssm.tf]:
```suggestion
  type  = "String"
```

File: modules/rds/ssm.tf [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  value = module.db.db_instance_address
+}
+
+resource "aws_ssm_parameter" "db_instance_name" {
+  name  = "/${var.name}/db_instance_name"
+  type  = "StringList"
```
Comment (ulises-jeremias) [inline modules/rds/ssm.tf]:
```suggestion
  type  = "String"
```

File: modules/rds/ssm.tf [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  value = module.db.db_instance_name
+}
+
+resource "aws_ssm_parameter" "db_instance_port" {
+  name  = "/${var.name}/db_instance_port"
+  type  = "StringList"
```
Comment (ulises-jeremias) [inline modules/rds/ssm.tf]:
```suggestion
  type  = "String"
```

File: modules/rds/ssm.tf [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  value = module.db.db_instance_port
+}
+
+resource "aws_ssm_parameter" "db_instance_master_user_secret_arn" {
+  name  = "/${var.name}/db_instance_master_user_secret_arn"
+  type  = "StringList"
```
Comment (ulises-jeremias) [inline modules/rds/ssm.tf]:
```suggestion
  type  = "String"
```

File: modules/rds/ssm.tf [hunk 5]
```diff
@@ -0,0 +1,23 @@
+resource "aws_ssm_parameter" "db_instance_address" {
+  name  = "/${var.name}/db_instance_address"
```
Comment (ulises-jeremias) [inline modules/rds/ssm.tf]: also, can we remove the `db_instance_` from all the names?

**Review by ulises-jeremias — COMMENTED** [review]
just some requests, the rest looks great!

File: modules/rds/ssm.tf (hunk 5 above)
Comment (AgusPk) [reply modules/rds/ssm.tf]: sure! should I leave db_address or just address?

Comment (ulises-jeremias) [reply modules/rds/ssm.tf]: addres 😊

Comment (AgusPk) [reply modules/rds/ssm.tf]: done!

**Review by ulises-jeremias — APPROVED** [review]
🔥

**Pull request merged**